pkg/reconciler/cmd/start.go

```diff
@@ -40,7 +47,9 @@ type ArgoEventsControllerOpts struct {
 }
 
 func Start(eventsOpts ArgoEventsControllerOpts) {
+
 	logger := logging.NewArgoEventsLogger().Named(eventbus.ControllerName)
+	log.SetLogger(controllerZap.New(controllerZap.UseDevMode(false)))
```

Comment:
```golang
import "github.com/go-logr/zapr"

log.SetLogger(zapr.NewLogger(logger.Desugar()))
```

Reply:
This change cannot be made b/c you are already concurrently using the `go.uber.org/zap` library. It's confusing but the k8s controller logging paradigm specifically requires `sigs.k8s.io/controller-runtime/pkg/log/zap` or it will throw an error.

Reply:
Have you tried that? 

The reason I suggested that because it will make the log format consistent for both controller logs and others (ex. dev mode).